setup.cfg

```diff
@@ -16,33 +16,28 @@ version = 2022.11.19
 #            versions that are no longer supported. This may break installing
 #            older versions of packages that depend on this meta-package!
 [options]
-python_requires = >=3.5
+python_requires = >=3.7
```

Comment:
So you think the `Important: ...` comment right above is incorrect? I think that's indeed the case (as long as `python_requires` gets bumped, users of older Python versions should get an older version of this meta-package), so let's edit that comment.

Reply:
Sorry for overlooking that comment. I haven't verified it, but it seems obvious to me that someone on py<3.7 would be even able to install a package that says `python_requires >= 3.7`.

I just thought it'd be worth to stop carrying around cruft for very old python versions anymore (we could even remove 3.7, it's EOL as of this month), but if you want to be sure, I can re-add them.

Reply:
> can re-add them.

No need to re-add, they're already included. The condition is `>= 3.7`, so 3.7 is there.